README.md

```diff
@@ -26,36 +24,80 @@ Latest version of Subfinder installed, for instructions see https://github.com/p
-4. Assign youruser, email and yourpass
+```
+1. git clone https://gitlab.eng.vmware.com/redteam/asfv2.git /opt/asf
+2. cd /opt/asf/
+3. Generate a .env.prod file or move from backup.env.prod and make necessary changes. 
+Run `./setup.sh`
+```
+
+MongoDB is required for functioning of alerting or reporting. 
+
+If you choose to run your own mongodb instance you may use the below command
+
+```
+docker run -dp 27017:27017 -v local-mongo:/data/db --name local-mongo --restart=always -e MONGO_INITDB_ROOT_USERNAME=<<>> -e MONGO_INITDB_ROOT_PASSWORD=<<>> mongo
+```
+
+
+And update .env.prod with following details:
+
+```
+MONGO_USER=admin
+MONGO_PASSWORD=
+MONGO_URL=
+MONGO_PORT=27017
+```
+
+
+Once the installation is completed ASF will run as service on port 2021, access by browsing to http://127.0.0.1:2021
 
-Once the installation is completed ASF will be available as a service on `http://127.0.0.1:2021`
 
 ### Security
 
 ASF is not meant to be publicly exposed, assuming you install it on a cloud provider or even on a local instance, we recommend to access it using port forwarding through SSH, here is an example:
 
-`ssh -i "key.pem" -L 8080:127.0.0.1:8080 user@yourhost` - For ASF GUI
+`ssh -i "key.pem" -L 2021:127.0.0.1:2021 user@yourhost` - For ASF GUI
 
-`ssh -i "key.pem" -L 9045:127.0.0.1:9045 user@yourhost` - To access Graylog2 Panel
 
 Then open your browser and go to: 
 
-`http://127.0.0.1:8080` - For ASF - user:youruser pass:yourpass (provided in initial setup)
+`http://127.0.0.1:2021` - For ASF - user:youruser pass:yourpass (provided in initial setup)
+
+
+###### Social Login 
+
+Additionally you may configure social login by enabling the right social login feature in .env.prod 
 
-`https://127.0.0.1:9045` - For Graylog2 - user:admin pass:admin #Change it in /graylog/docker-compose.yaml
+```
+#LOGIN CONFIGURATIONS
+LOGIN_FORM=True
+SOCIAL_AUTH_GOOGLE_ENABLED=True (False)
+SOCIAL_AUTH_GITHUB_ENABLED=True (False)
+DJANGO_SAML2_ENABLED=True (False)
+```
+Navigating to admin panel http://127.0.0.1:2021/admin to add the social app. 
 
-Graylog2 requires a few steps to start receiving logs from ASF: 
+`Note that if social login is enabled and not configured in admin panel ASF will not display the login page - to revert simply disable the Social Login in .env.prod`
 
-Once logged in, go to System/"Content Packs" and import the Content Pack located at /opt/asf/tools/graylog/content_pack_ASF.json, click on the "Upload" button and you should see "Basic" reflected in the "Select Content Packs" section, click on "Basic", make sure the "ASF" radio button is selected and hit the "Apply content" button, this will create the Global input to parse JSON logs and related extractors. 
 
-![Graylog2 Inputs Example](images/Graylog_content_pack.jpg)
+###### Jira Integration
 
-Now you are ready to receive logs from ASF and setup your streams / alerts / dasboards ! 
+ASF support Jira integration and ticketing to enable Jira set the following values in .env.prod:
```

Comment:
My suggestion "To enable Jira integration and ticketing, ASF supports setting the following values in the .env.prod file"